Makefile

```diff
@@ -1,4 +1,4 @@
-ENVS := flake8,py27,py36
+ENVS := flake8,py27,py36,py37
```

Comment:
The reason why I haven't added this is that on my RHEL7 laptop I don't have a working Python 3.7 environment, and adding this makes Tox fail. Maybe there is a way to no fail on `InterpreterNotFound`.